core/src/main/java/org/eclipse/hono/config/ProtocolAdapterProperties.java

```diff
@@ -229,4 +231,15 @@ public Duration getTenantIdleTimeout() {
     public void setTenantIdleTimeout(final Duration tenantIdleTimeout) {
         this.tenantIdleTimeout = Objects.requireNonNull(tenantIdleTimeout);
     }
+
+    /**
+     * Gets the configured mappers for this adapter
+     * <p>
+     * The default value of this property is an empty hashMap. Ths results in not having custom mapping at all.
+     *
+     * @return HashMap of mapper endpoints
+     */
+    public HashMap<String, MapperEndpoint> getMapperEndpoints() {
```

Comment:
how do you set these?

Reply:
Did not run it yet but was hoping for the spring-boot to provide this via the properties file

Reply:
you better try this out :-)

Reply:
I tried it out and it works :)
Following configuration results in 2 mapperEndpoints being configured in the (mqtt) adapter:
```
  mqtt:
    .....
    mapperEndpoints:
      test:
        host: host
        port: 123
        uri: uri
      test2:
        host: host2
        port: 321
        uri: uri2
```

Reply:
Now you will only need to find a way to document this :-)

Reply:
new day, new insights. Shouldn't have added the underscore in `MAPPER_ENDPOINTS` but left it as is: `MAPPERENDPOINTS`. Now it's wrorking. Documentation also updated.